traits/trait_base.py

```diff
@@ -203,15 +201,15 @@ def strx(arg):
 #  Constants:
 # -------------------------------------------------------------------------------
 
-StringTypes = (str, six.text_type, int, LONG_TYPE, float, complex)
+StringTypes = (str, six.text_type, int, float, complex)
 
 # -------------------------------------------------------------------------------
 #  Define a mapping of coercable types:
 # -------------------------------------------------------------------------------
 
 # Mapping of coercable types.
 CoercableTypes = {
-    LONG_TYPE: (11, LONG_TYPE, int),
+    int: (11, int),
```

Comment:
I'm not sure whether this actually needs to be here: `CoercableTypes` doesn't get used anywhere that I can find (and the spelling is bugging me, too, but that's a rant for another day).

Reply:
Ah, `CoercableTypes` *does* get used in `TraitCoerceType`. The entry for `int` is not needed here. I'll remove it.